server-templates/api.mustache

```diff
@@ -83,6 +85,7 @@ public class {{classname}}  {
 
   private final {{classname}}Service service;
 
+  @Inject
```

Comment:
If you turn `RealmContext` into request-scoped, a nice benefit that comes for free is that you could inject it directly into the REST resources generated from the open-api spec. This way, the realm is immediately available and injected everywhere. E.g. the /config endpoint could become:

```java
  public Response getConfig(
      @QueryParam("warehouse") String warehouse,
      @Context RealmContext realmContext,
      @Context SecurityContext securityContext) {
    return service.getConfig(warehouse, realmContext, securityContext);
  }
```